lib/ecco/row_event_listener.rb

```diff
@@ -5,8 +5,11 @@ module Ecco
   class RowEventListener < EventListener
     ROW_EVENTS = [
       EventType::WRITE_ROWS,
+      EventType::EXT_WRITE_ROWS,
       EventType::UPDATE_ROWS,
+      EventType::EXT_UPDATE_ROWS,
       EventType::DELETE_ROWS,
+      EventType::EXT_DELETE_ROWS,
```

Comment:
Shouldn't we have specs for both then? You added to the array here, but replaced them in the specs (e.g. WRITE_ROWS -> EXT_WRITE_ROWS)